tests/nat/conftest.py

```diff
@@ -151,6 +151,10 @@ def apply_global_nat_config(duthost):
     after test run cleanup DUT's NAT configration
     :param duthost: DUT host object
     """
+    status = duthost.get_feature_status()
+    if 'nat' not in status:
```

Comment:
return type is a tuple for this function.  check should be 'nat' not in status[0]

Reply:
Thanks, you are right.